python/servo/package_commands.py

```diff
@@ -132,31 +133,21 @@ class PackageCommands(CommandBase):
                      default=None,
                      help='Package using the given Gradle flavor')
     @CommandBase.common_command_arguments(build_configuration=False, build_type=True)
-    def package(self, build_type: BuildType, android=None, target=None, flavor=None, with_asan=False):
-        if android is None:
-            android = self.config["build"]["android"]
-        if target and android:
-            print("Please specify either --target or --android.")
-            sys.exit(1)
-        if not android:
-            android = self.setup_configuration_for_android_target(target)
-        else:
-            target = self.config["android"]["target"]
-
-        self.cross_compile_target = target
+    @CommandBase.allow_target_configuration
+    def package(self, build_type: BuildType, flavor=None, with_asan=False):
         env = self.build_env()
-        binary_path = self.get_binary_path(build_type, target=target, android=android, asan=with_asan)
+        binary_path = self.get_binary_path(build_type, asan=with_asan)
         dir_to_root = self.get_top_dir()
         target_dir = path.dirname(binary_path)
-        if android:
-            android_target = self.config["android"]["target"]
-            if "aarch64" in android_target:
+        if isinstance(self.target, AndroidTarget):
```

Comment:
It might be nice to add a helper for this in the base class `def is_android(...)`.